README.npm.md

```diff
@@ -281,6 +281,29 @@ For token based auth, this would be provided like so:
 }
 ```
 
+For bearer base auth (e.g: BitBucket Server PAT), this would be provided like so:
+```json
+{
+  "private": [
+    {
+      "method": "GET",
+      "path": "/myself",
+      "origin": "https://${HOST}",
+      "auth": {
+        "scheme": "bearer",
+        "bearer": "${BEARER}"
+      }
+    },
+  ],
+  "public": [
+    {
+      "method": "any",
+      "path": "/*"
+    }
+  ]
+}
```

Comment:
This isn't really relevant to this change because it's copying what's already there, but it's weird that we list the `public` block in each of these examples..

Reply:
tbh this is a copy paste as u said... don't even know what's the diff between public and private.. or don't remember if i was told in the past :) but i'll just leave it as is and leave it up to ur team to update the docs as needed :)